governance/mixed.go

```diff
@@ -0,0 +1,188 @@
+package governance
+
+import (
+	"github.com/klaytn/klaytn/blockchain/types"
+	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/consensus/istanbul"
+	"github.com/klaytn/klaytn/params"
+	"github.com/klaytn/klaytn/storage/database"
+)
+
+type MixedEngine struct {
+	// Subordinate engines
+	defaultGov *Governance
+}
+
+func newMixedEngine(config *params.ChainConfig, db database.DBManager, doInit bool) *MixedEngine {
+	e := &MixedEngine{}
+
+	// Setup subordinate engines
+	if doInit {
+		e.defaultGov = NewGovernanceInitialize(config, db)
+	} else {
+		e.defaultGov = NewGovernance(config, db)
+	}
+	return e
+}
+
+// Most users are encouraged to call this constructor
```

Comment:
```suggestion
// NewMixedEngine instantiate a new MixedEnginestruct. Developers are encouraged to call this constructor in most cases.
```

Reply:
Fixed in https://github.com/klaytn/klaytn/pull/1298/commits/4cc46318076f6ec2fb03ebf1ea4f13108ee51a6d.